eZ/Bundle/EzPublishCoreBundle/Imagine/AliasGenerator.php

```diff
@@ -118,12 +127,17 @@ public function getVariation(Field $field, VersionInfo $versionInfo, $variationN
         }
 
         try {
+            $resolvedPath = $this->ioResolver->resolve($originalPath, $variationName);
+            $image = $this->imagine->open($resolvedPath);
```

Comment:
As we discussed on the slack: we need to use a path to the generated image instead of URL. Downloading image over the network have a significant impact on the performance.

Reply:
> s we discussed on the slack: we need to use a path to the generated image instead of URL. Downloading image over the network have a significant impact on the performance.

True, I'll fix it today the way we agreed.